crates/rspack/tests/tree-shaking/simple-namespace-access/expected/main.js

```diff
@@ -35,8 +35,7 @@ __webpack_require__.d(__webpack_exports__, {
 'use strict';
 __webpack_require__.r(__webpack_exports__);
 __webpack_require__.d(__webpack_exports__, {
-  'test': function() { return test; },
-  'ccc': function() { return ccc; }
+  'test': function() { return test; }
```

Comment:
https://github.com/web-infra-dev/rspack/pull/3895/files#diff-151f123db6910c5557f26eca501183dd1847381ba0e7905ca22e266e834fc848R7, only `xxx.test` is used, so ccc should not be exported.